manifests/a/Atlassian/Companion/2.0.1.0/Atlassian.Companion.locale.en-US.yaml

```
@@ -0,0 +1,13 @@
+# Automatically updated by the winget bot at 2023/Dec/04
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: Atlassian.Companion
+PackageVersion: 2.0.1.0
+PackageLocale: en-US
+Publisher: Atlassian, Inc.
+PackageName: Atlassian Companion
+License: Proprietary
+Copyright: Copyright © 2023 Atlassian
+ShortDescription: The Atlassian Companion app enables users to edit Confluence files in their preferred desktop application, then save the file back to Confluence automatically.

```

```suggestion
ShortDescription: The Atlassian Companion app enables users to edit Confluence files in their preferred desktop application, then save the file back to Confluence automatically. 
```
Test change please ignore.